vivo-project/Vitro
# Add check for language files to smoke tests and remove references to …

…build.properties

**[JIRA Issue VIVO-1501](https://jira.duraspace.org/browse/VIVO-1501)**: ConfigurationPropertiesSmokeTests raises outdated warning on multi-language startup

# What does this pull request do?
This PR fixes a problem where a warning was raised during startup anytime you started VIVO and are using selectable internationalization.

# What's new?
Adds a test to the configuration smoke tests to check if language files exist at the top webapp level that correspond to all the selectable languages. If the correct language files don't exist, a warning is thrown during startup. 

Also removes a test that used to check if languages were specified in build.properties, which no longer exists and adjusts a test that checked for languages in build.properties to use the languages specified by selectableLanguages in runtime.properties.

# How should this be tested?
Replicate the issue by deploying VIVO with the vivo-languages package included and adding a couple languages to the selectable languages setting in runtime.properties. A warning is thrown about languages not being included in build.properties (which doesn't exist anymore).

Try again with this PR. Above warning should not appear.
Try adding a valid language that isn't included in vivo-languages (e.g. fr_FR) to selectableLanguages in runtime.properties and check that warning is raised during startup. You could also try an invalid locale (e.g. fo_BA). The same warning will be raised and you'll also get a warning from LocaleUtils that it's an invalid locale.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* Does this change require documentation to be updated? 
* Does this change add any new dependencies? 
* Does this change require any other modifications to be made to the repository? 
* Could this change impact execution of existing code?

# Interested parties
@KitioFofack
